src/big/big_float.cr

```diff
@@ -217,11 +243,21 @@ struct BigFloat < Float
     mpf
   end
 
+  # Returns an inspected representation of `self`.
```

Comment:
Returns a human-readable representation of self.

Reply:
Done 👍